src/app/about/page.tsx

```diff
@@ -9,63 +11,118 @@ export const metadata: Metadata = {
 
 export default function About() {
   return (
-    <div>
-      <h1>活動内容</h1>
-      <h2>競技プログラミング</h2>
-      <h3>どんなことをしているの？</h3>
-      <p>
-        国際大学対抗プログラミングコンテスト(ICPC)に参加し、良い成績を上げるための活動を行っています。
-        具体的には、ICPCなどの各種プログラミングコンテストへの参加、週二回の勉強会などを行っています。
-      </p>
-      <h3>競技プログラミングってそもそも何？</h3>
-      <p>
-        皆さんご存じの通り、コンピュータはとても高速に計算してくれます。
-        しかし、その速さには限界があり、世界中の多くの問題はそのままコンピュータで解くと数万年・数億年単位の時間を要すると言われています。
-        そこで、こういった問題の答えを高速かつ正確に出すための工夫を行って、プログラムとして記述するのが競技プログラミングです。
-      </p>
-      <p>
-        工学部情報工学科のパンフレットに記載していただきました！
-        例題も掲載されているので是非ご覧ください。
-        <a
-          href='http://www.ics.saitama-u.ac.jp/content/uploads/pamphlet/ics_pamphlet.pdf'
-          target='_blank'
-        >
-          情報工学科パンフレット
-        </a>
-      </p>
-      <img src='/images/ics-pamphlet.png' alt='情報工学科パンフレット' />
-      <div>
-        <Link href='/achievements'>
-          <strong>過去の成績|Maximum</strong>
-        </Link>
-      </div>
-      <h2>Web</h2>
-      <h3>講習</h3>
-      <p>
-        WebサイトやWebサーバーを制作・構築・運用するための基礎を一通り学びます。
-        講習ではHTML,CSS,JabaScriptを触ります。Webに関しては
-        <strong>アウトプットこそ正義</strong>
-        なので「勉強する」よりも「作りながらその都度調べる」というフローで効率の良い学習ができるようサポートします。
-      </p>
-      <h3>サイト制作・アプリ制作</h3>
-      <p>
-        一通り基礎を学び終えたら、実際にX（旧twitter）やNoteのような投稿ができるサービスを自分たちで作ってもらいます。
-      </p>
-      <h3>パフォーマンスチューニング</h3>
-      <p>
-        Webの醍醐味は「制作」だけではありません。
-        この世には「遅すぎる」や「セキュリティ的に危険」などの様々な問題を抱えたサイトがあります。
-        こんなサイトを高速化したり安全に書き換えたりなどするのも、もちろんWebの範疇です。
-      </p>
-      <p>
-        一見競技性のないWebの分野でも、こういった問題を解決する力を競う大会がいくつか開催されており、
-        そこに出場することを目標にトレーニングもしています。
-      </p>
-      <p>詳しく知りたい方は大会参加記を見てみてください。</p>
-      <div>
-        <Link href='/achievements'>
-          <strong>過去の成績|Maximum</strong>
-        </Link>
+    <div className={style.container}>
+      <Header />
+        <img className={style.hero} src='/heros/hero-about.png' alt='hero' />
```

Comment:
Figma に書いてある通り、 hero はわかめぼかし + タイトル h1 で

![image](https://github.com/saitamau-maximum/public-website/assets/26807394/a7bf06db-32b0-4016-a641-2a721a801a84)

Reply:
ただ時間もないので TODO コメント残していったんそのままでいい気もしますね

```suggestion
        {/* TODO: Hero 画像とタイトルを分離する */}
        <img className={style.hero} src='/heros/hero-about.png' alt='hero' />
```